specification/azurestack/resource-manager/Microsoft.AzureStack/stable/2017-06-01/Product.json

```diff
@@ -143,6 +143,147 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.AzureStack/registrations/{registrationName}/products/_all/GetProducts": {
```

Comment:
We need this request to be POST, to add Configuration body
And RP SDK that we use for routing and underlying AzureStack configurations prohibit "Post" calls to "GetResource"
So that's a hack-around limitations we have on system

Reply:
@ryansbenson  We intend to deprecate the GET call on the product and replace it with this new POST call GetProducts, Can we simply delete the existing GET Product call? For this swagger , we have not published any SDK or powershell/CLI yet, so the impact is minimal . I am inclined to delete the existing GET call to remove the confusion. What do you suggest?

Reply:
We discussed internally with team - for now we shouldn't delete GET requests entirely. In time it will most likely transform to behave differently from POST.